app/demo-stm32h7-nucleo/app-h743.toml

```diff
@@ -279,19 +279,20 @@ af = 4
 # driver = "ltc4306"
 # address = 0b1001_010
 
-[config.spi.spi3]
-controller = 3
+[config.spi.spi1]
```

Comment:
Worth adding a cautionary note here about SPI3?  (It may not be...)  FWIW, here is the comment that I had when carefully avoiding TRACESWO:

```rust
                            //
                            // It may seem reasonable -- especially given the
                            // groupings and silkscreen on CN7 -- to use PB3
                            // for SPI3_SCK. However, this pin is TRACESWO on
                            // AF0:  if we configure this pin for SPI, ITM
                            // will no longer work (and with a failure mode
                            // that can be difficult to debug).  We instead
                            // opt to use PC10 for SPI3_SCK, which is also
                            // easily accessible:  it's pin 6 on CN8.
                            //
```

Again, not sure if this is worth it -- it's just so painful to get wrong.

Reply:
I don't think the note is necessary, because anyone following the SPI labels on the board won't consider mapping it to PB3.